recipes/podofo/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class OatppSwaggerConan(ConanFile):
+    name = "podofo"
+    license = "GNU-2"
```

Comment:
Do you mean `GPL-2.0` or `GPL-2.0-or-later`?
Check spdx.org for the correct tags

Reply:
:) GPL-2.0